src/box/sql/build.c

```diff
@@ -59,6 +59,75 @@
 #include "box/user.h"
 #include "box/constraint_id.h"
 #include "box/session_settings.h"
+#include "mpstream/mpstream.h"
+
+/**
+ * Encode format as entry to be inserted to _space on @region.
+ *
+ * @param parse Parse context.
```

Comment:
There is no `parse` parameter. Also you can probably make `def` const (same in `encode_opts()`).

Reply:
Hi! Thank you for the review! I decided to drop some patches and move some patches to another PR. This change is from dropped patches.